tests/test_views.py

```diff
@@ -0,0 +1,144 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+from taxi.models import Manufacturer, Car, Driver
+from taxi.forms import ManufacturerSearchForm, CarSearchForm, DriverSearchForm
```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports)